paddle/fluid/distributed/auto_parallel/test/CMakeLists.txt

```diff
@@ -15,4 +15,7 @@ cc_test(
 
 cc_test_old(dist_mapper_test SRCS dist_mapper_test.cc DEPS phi)
 
-cc_test_old(spmd_rule_test SRCS spmd_rule_test.cc DEPS spmd_rule)
+cc_test_old(spmd_rule_test SRCS spmd_rule_test.cc DEPS spmd_rules)
+
+cc_test_old(elementwise_spmd_rule_test SRCS ./elementwise_spmd_rule_test.cc
```

Comment:
where is elementwise_spmd_rule_test

Reply:
Added elementwise_spmd_rule_test.cc back, the test cases in elementwise_spmd_rule_test.cc is less than test_elementwise_rule.py

Reply:
Done